pandas/tests/test_strings.py

```diff
@@ -1210,7 +1210,6 @@ def test_empty_str_methods(self):
         empty_str = empty = Series(dtype=str)
         empty_int = Series(dtype=int)
         empty_bool = Series(dtype=bool)
-        empty_list = Series(dtype=list)
```

Comment:
actually, this is the *same* as ``Series(dtype=str)``, so just change that one to ``Series(dtype=object)`` and done